src/server/components/dashboard/listsItems/listItemsIndexController.ts

```diff
@@ -171,13 +174,27 @@ export async function listItemsIndexController(
   }
 }
 
-function annualReviewBannerToggles(list: ListWithJsonData, listItems: IndexListItem[]) {
+async function annualReviewBannerToggles(list: ListWithJsonData, listItems: IndexListItem[]) {
   const emailsSent = listItems?.some((listItem) => {
     return SummaryHelpers.annualReviewEmailsSent(list, listItem?.history);
   });
-  return {
-    oneMonthWarning: displayOneMonthAnnualReviewWarning(list),
-    emailsSent,
-    unpublishWarning: displayUnpublishWarning(list, listItems),
-  };
+
+  const unpublishWarning = await displayUnpublishWarning(list);
+
+  if (unpublishWarning) {
+    return {
+      unpublishWarning,
+    };
+  }
+
+  if (emailsSent) {
+    return { emailsSent };
+  }
```

Comment:
This could be rewritten as
```suggestion
switch (true) {
  case Boolean(unpublishWarning):
    return { unpublishWarning };
  case Boolean(emailsSent):
    return { emailsSent };
  default:
    return {};
}
```

But not a required change, just looks a bit weird 🤷

Reply:
looks weird because it is.. I'm trying to fix an issue with multiple banners showing up without changing too much, but I think I will just change how the function works

before annualReviewBannerToggles would return an object containing the above 

```.ts
{
 unpublishWarning: {..},
 emailsSent: {..},
 displayOneMonthWarning: {..} 
}
```

issue described in the ticket was multiple banners showing up at once. `annualReviewBannerToggles` should just return one unnested banner object (rather than rely on display: true/false).